### ISSUE TITLE
No Phone App

### ISSUE THREAD

**Issue by jschultzGetHub** [matejdro/WearMusicCenter#22]
Installed Media Center using Wear Installer.  App was loaded and added to App shortcuts.  When I try to open it I get a message "Install Phone app" and a link to "OPEN PHONE PLAY STORE".  It's galaxy Classic 6 watch Wear OS ver 4.0. I have two phone apps installed- Samsung Phone and Google Phone.  Can offer any suggestions?  TIA, Jim S.

**Comment by matejdro** [comment]
You also have to install Music Center phone app.

**Comment by jschultzGetHub** [comment]
Sorry but I'm not a developer. Searched and could not find the MC phone app.  Appreciate it if you could point me in the right direction...

**Comment by matejdro** [comment]
You can get it here: https://github.com/matejdro/WearMusicCenter/releases/tag/1.9.2

**Comment by jschultzGetHub** [comment]
Yes, that's what I installed. Any reason why the phone app is missing?

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by matejdro** [comment]
There are two apps on that link, watch app and phone app.
